doc/modules/linear_model.rst

```diff
@@ -313,15 +314,75 @@ is correct, i.e. that the data are generated by this model.
 They also tend to break when the problem is badly conditioned
 (more features than samples).
 
-.. figure:: ../auto_examples/linear_model/images/sphx_glr_plot_lasso_model_selection_001.png
-    :target: ../auto_examples/linear_model/plot_lasso_model_selection.html
+.. figure:: ../auto_examples/linear_model/images/sphx_glr_plot_lasso_lars_ic_001.png
+    :target: ../auto_examples/linear_model/plot_lasso_lars_ic.html
     :align: center
     :scale: 50%
 
+Mathematical details
+""""""""""""""""""""
```

Comment:
This renders as `1.1.3.1.2.1. Mathematical details`. Could we skip the chapter numbering here?